docs/building.md

```diff
@@ -6,31 +6,32 @@ If you come across a disfunctional setup, please report it by open an issue or p
+   |     PARTHENON\_DISABLE\_HDF5 | OFF                        | Option | HDF5 is enabled by default if found, set this to True to disable HDF5 |
+   |      PARTHENON\_DISABLE\_MPI | OFF                        | Option | MPI is enabled by default if found, set this to True to disable MPI |
+   |   PARTHENON\_DISABLE\_OPENMP | OFF                        | Option | OpenMP is enabled by default if found, set this to True to disable OpenMP |
+   |   ENABLE\_COMPILER\_WARNINGS | OFF                        | Option | Enable compiler warnings |
+   |        TEST\_ERROR\_CHECKING | OFF                        | Option | Enables the error checking unit test. This test will FAIL |
+   |    TEST\_INTEL\_OPTIMIZATION | OFF                        | Option | Test intel optimization and vectorization |
+   |    CHECK\_REGISTRY\_PRESSURE | OFF                        | Option | Check the registry pressure for Kokkos CUDA kernels |
+   |               BUILD\_TESTING | ON                         | Option | Multi-testing enablement |
+   | PARTHENON\_DISABLE\_EXAMPLES | OFF                        | Option | Toggle building of examples, if regression tests are on, drivers needed by the tests will still be built |
+   |   ENABLE\_INTEGRATION\_TESTS | ${BUILD\_TESTING}          | Option | Enable integration tests |
+   |    ENABLE\_REGRESSION\_TESTS | ${BUILD\_TESTING}          | Option | Enable regression tests |
+   |      NUM\_MPI\_PROC\_TESTING | 4                          | String | Number of MPI ranks used for MPI-enabled regression tests |
+   |   NUM\_GPU\_DEVICES\_TESTING | ${NUM\_MPI\_PROC\_TESTING} | String | Number of GPUs to use if buildt with kokkos_ENABLE_CUDA |
+   |  REGRESSION\_GOLD\_STANDARD\_VER | #                      | Int    | Version of current gold standard file used in regression tests. Default is set to latest version matching the source. |
+   | REGRESSION\_GOLD\_STANDARD\_HASH | SHA512=...             | String | Hash value of gold standard file to be downloaded. Used to ensure that the download is not corrupted. |
+   | REGRESSION\_GOLD\_STANDARD\_SYNC | ON                     | Option | Create `gold_standard` target to download gold standard files |
+   |          ENABLE\_UNIT\_TESTS | ${BUILD\_TESTING}          | Option | Enable unit tests |
+   |               CODE\_COVERAGE | OFF                        | Option | Builds with code coverage flags |
+   |       CMAKE\_INSTALL\_PREFIX | machine specific           | String | Optional path for library installation |
+   |                 Kokkos\_ROOT | unset                      | String | Path to a Kokkos source directory (containing CMakeLists.txt) |
+   |  PARTHENON\_IMPORT\_KOKKOS | ON/OFF                       | Option | If ON, attempt to link to an external Kokkos library. If OFF, build Kokkos from source and package with Parthenon |
+   |          BUILD\_SHARED\_LIBS | OFF                        | Option | If installing Parthenon, whether to build as shared rather than static |
```

Comment:
cleaning up the markdown is a much needed improvement. Thanks for taking the time.